frontend/src/modules/entitydetails/selectors.js

```diff
@@ -51,6 +54,34 @@ export const getTranslationForSelectedEntity: Function = createSelector(
 );
 
 
+export function _isReadOnlyEditor(
+    entities: Entities,
+    params: NavigationParams,
+    user: UserState,
+): boolean {
+    const selectedEntity = entities.find(element => element.pk === params.entity);
```

Comment:
I think this should use the `getSelectedEntity` selector instead (in `modules/entitieslist` iirc). I'm not entirely sure though, it might actually make the code more complex, so I'll let you judge that.